perfkitbenchmarker/analyze.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
I think the cleanest solution is to make the object storage benchmark a package, with two modules inside of it: the regular benchmark module, and this analysis module. I implemented that, but the change added a bunch of boring changes to this PR, just updating names. I can merge this as-is and immediately submit that as my next PR, or I could add the extra code to this PR. The new code wouldn't be complicated to review, there would just be more of it to look at. What do you think?